Jupyter/QuantBook.cs

```diff
@@ -488,12 +488,14 @@ private PyObject Indicator(IndicatorBase<IndicatorDataPoint> indicator, IEnumera
 
             selector = selector ?? (x => x.Value);
 
-            foreach (var bar in history)
+            foreach (var slice in history)
             {
-                var value = selector(bar);
-                indicator.Update(bar.EndTime, value);
+                foreach(var bar in slice.Values)
+                {
+                    var value = selector(bar);
```

Comment:
I think sometimes `bar` here may be a `List<Tick>` -- see `SliceExtensions.PushThrough` to see if you need to do something similar to handle that case.

Reply:
`SliceExtensions.PushThrough` is just perfect. Updating the PR... 
Thanks!